src/apps/authentication/__test__/components/UserOrganisationDetails.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {render} from "../../../../test-utils";
+import {UserOrganisationDetails} from "../../components";
+import {testAPIOrgList} from "../test_data";
+
+type userOrganisationDetailsProps = React.ComponentProps<typeof UserOrganisationDetails>;
+
+const baseProps: userOrganisationDetailsProps = {
+    orgs: testAPIOrgList
+};
+
+function renderUI(props: Partial<userOrganisationDetailsProps> = {}) {
+    return render(<UserOrganisationDetails {...baseProps} {...props} />);
+}
+
+describe("UserOrganisationDetails", () => {
+    it("should match snapshot", () => {
+        const {container} = renderUI(baseProps);
+
+        expect(container).toMatchSnapshot();
+    });
+});
```

Comment:
Apart from doing snapshot test, you can add one test to validate current org name is displayed or not

Reply:
Hi Rakesh, we didn't add new tests as this is a view page and the details we be covered as part of snapshot.Hope that is fine